src/containers/shared/components/Transaction/NFTokenAcceptOffer/types.ts

```diff
@@ -0,0 +1,17 @@
+import { TransactionCommonFields } from '../types'
+
+export interface NFTokenAcceptOffer extends TransactionCommonFields {
+  NFTokenSellOffer?: string
+  NFTokenBuyOffer?: string
+  NFTokenBrokerFee?:
+    | string
+    | { currency: string; value: string; issuer: string }
```

Comment:
Is there an `Amount` or `IssuedCurrencyAmount` type somewhere in the repo? (That's what this type is)

Reply:
These types exist in Transactions/XChainAddAttestation. @ckniffen These should probably moved to general types file in Transactions in future.

Reply:
so it get turned into a number by formatAmount.  Maybe there should be a type of FormattedAmount